coalib/parsing/Globbing.py

```diff
@@ -299,33 +306,35 @@ def relative_wildcard_glob(dirname, pattern):
     return result
 
 
-def relative_flat_glob(dirname, basename):
+def relative_flat_glob(dirname, basename, ignored_globs=None):
```

Comment:
don't understand that. If you don't need it, then you don't need, am I wrong? :D